pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -195,25 +196,30 @@ func (og *operationGenerator) GenerateVolumesAreAttachedFunc(
 				}
 			}
 		}
-		return nil
+		return nil, nil
+	}
+
+	return volumetypes.GeneratedOperations{
+		OperationFunc: volumesAreAttachedFunc,
+		CompleteFunc:  util.OperationCompleteHook("<n/a>", "verify_volumes_are_attached_per_node"),
```

Comment:
nit: Explicitly add `EventRecorderFunc: nil, // reason why nil` so that folks reading this code in the future know this was left nil on purpose and why.